addons/addons.json

```diff
@@ -153,7 +153,6 @@
   "big-save-button",
   "workspace-dots",
   "asset-conflict-dialog",
-  "editor-cleanup-plus",
```

Comment:
Please ensure that no `addon.json` items have been accidentally moved or deleted.